Surface_mesh/examples/Surface_mesh/CMakeLists.txt

```diff
@@ -6,9 +6,14 @@ project( Surface_mesh_Examples )
 
 cmake_minimum_required(VERSION 2.8.11)
 
-if(NOT POLICY CMP0070 AND POLICY CMP0053)
+if(NOT POLICY CMP0070 AND  POLICY CMP0053)
   # Only set CMP0053 to OLD with CMake<3.10, otherwise there is a warning.
   cmake_policy(SET CMP0053 OLD)
+  elseif(POLICY CMP0070 AND  POLICY CMP0053)
+    cmake_policy(SET CMP0053 NEW)
+endif()
+if(POLICY CMP0070 AND  POLICY CMP0071)
+    cmake_policy(SET CMP0071 OLD)
```

Comment:
- That is strange to test `CMP0070`. You can simplify to:
```cmake
if(POLICY CMP0071)
```

- Why the old behavior for `CMP0070`?! We do not have any generated file on which we want to run old. Set it to `NEW`. That will avoid issues the day `CMP0071` stopped being supported.